core-blocks/test/fixtures/core__preformatted.json

```diff
@@ -8,11 +8,12 @@
                 "Some ",
                 {
                     "type": "em",
-                    "children": "preformatted"
+                    "children": [ "preformatted" ]
                 },
                 " text...",
                 {
-                    "type": "br"
+					"type": "br",
```

Comment:
Funny formatting? Running `npm run fixtures:regenerates` introduces a number of changes locally.